gslib/commands/cp.py

```diff
@@ -282,82 +276,79 @@
     Copying file://obj [Content-Type=text/plain]...
     Uploading   gs://your-bucket/obj:                                182 b/182 B
 
-    If the checksum didn't match the service would instead reject the upload and
-    gsutil would print a message like:
+    If the checksums don't match, the service rejects the upload and
+    gsutil prints the following message:
 
     BadRequestException: 400 Provided MD5 hash "VgyllJgiiaRAbyUUIqDMmw=="
     doesn't match calculated MD5 hash "7gyllJgiiaRAbyUUIqDMmw==".
 
-  Even if you don't do this gsutil will delete the object if the computed
```

Comment:
Do you think this automatic behavior doesn't warrant noting?

Reply:
I found it redundant, since the very first paragraph in the section says "If the checksums do not match, gsutil deletes the corrupted object and prints a warning message." In the following discussion, I didn't find any indication that specifying Content-MD5 contradicts that statement. In fact, the bullet points after this are variations on the assumptions that "gsutil deletes objects with mismatched checksums." So I didn't feel the need to reiterate that point here. If you feel that there's something missing here, let me know.

Reply:
Reviewing the text again, I agree with you!